docs/firststeps.rst

```diff
@@ -1,35 +1,129 @@
-First steps
+First Steps
 ===========
 
+This is an easy to follow tutorial that gets you up to speed with *CliMetLab*
+to access scientific climate data.
+It assumes you have basic Python programming knowledge.
+
+So, let's start with what *CliMetLab* does best:
+
 Getting data
 ------------
 
-There are two ways of loading data in *CliMetLab*, using :ref:`data
-sources <data-sources>` or :ref:`datasets <datasets>`.
+*CliMetLab* provides two main ways to access climate data
+
+1. Data Sources
+2. Datasets
 
 Data sources
 ^^^^^^^^^^^^
 
-Data sources implement various methods to access and decode data.
-When data are downloaded from a remote site, they are :ref:`cached
-<caching>` on the local computer.
+In *CliMetLab*, a *Data Source* refers to a local or remote storage server or data archive
+from where we can download or access files related to climate data.
+
+To get started, let us first import *CliMetLab* at the top of our Python notebook:
+
+.. code-block:: python
+
+    import climetlab as cml
+
+Now download
+`test.grib <https://raw.githubusercontent.com/ecmwf/climetlab/develop/docs/examples/test.grib>`_ (example GRIB file)
+to your project directory, or if you have ``wget`` command available, run
+following in your notebook:
+
+.. code-block:: python
+
+    !wget https://raw.githubusercontent.com/ecmwf/climetlab/develop/docs/examples/test.grib
+
+GRIB is a file format for storage and sharing of gridded meteorological data.
```

Comment:
We do assume some previous knowledge of the domain, so describing GRIB is not necessary. But it is nice to have this for completeness.